### PR TITLE
Added tracking for search.

### PR DESCRIPTION
**Pull request by dsjen**
Depends on https://github.com/edx/edx-analytics-dashboard/pull/473.

@dan-f, please review.

### CODE REVIEW

File: analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js [hunk 1]
```diff
@@ -540,6 +540,18 @@ define(function (require) {
                 getLastRequest().respond(200, {}, JSON.stringify(getResponseBody(1, 1)));
             });
 
+            it('triggers tracking event', function () {
```
Comment (dan-f) [inline analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js]: Nit: 'triggers _a_ tracking event'.

File: analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js [hunk 2]
```diff
@@ -540,6 +540,18 @@ define(function (require) {
...
+                var rosterView = getRosterView(),
+                    searchString = 'search string',
+                    triggerSpy = spyOn(rosterView.options.trackingModel, 'trigger');
+
+                executeSearch(searchString);
+                expectSearchedFor(searchString);
+                expect(triggerSpy).toHaveBeenCalledWith('segment:track', 'edx.bi.roster.searched', {
+                    category: 'search'
```
Comment (dan-f) [inline analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js:551]: Are we supposed to capture what was searched for?

Also, just curious, what's the purpose of `category` here? Seems a bit redundant with the track event name.

**Comment by dan-f** [comment]
A couple questions, but 👍  on the code!

File: analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js (hunk 2 above)
Comment (dsjen) [inline analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js:551]: I looked at the spec and it looks like "search" is what was agreed upon.  We can easily modify it later if something else is desired.  Good question though!

**Pull request merged**